INSTALL.OSX.md

```diff
@@ -2,70 +2,60 @@
-  1. `npm install -g bower`
+1. Run `make deps`, this will install both client and server dependencies.
 
-7. Run `npm install` so that it can grab the packages in packages.json.
-
-8. Run `composer install` to install all our PHP dependencies.
-
-9. Run `grunt install` which will run some more install commands,
-downloading and installing the required packages. You may run into an issue
-with Github's API ratelimiting your requests for composer install, [see this
-workaround](https://coderwall.com/p/kz4egw).
-  1. If you don't have the `grunt` command, do `npm install -g grunt-cli`
-
-10. Install compass. If you do not have any ruby gems installed, you can get away with
+1. Install compass. If you do not have any ruby gems installed, you can get away with
 `sudo gem update --system` and `sudo gem install compass`, installing the compass gem
 on top of Mac OSX's ruby.
   1. In general, you will want to use something like [rvm](rvm) to manage a separate version
   of ruby which does not interfere with OSX. If you use rvm, drop the `sudo` from
   the aforementioned commands.
 
-11. Set up the site as an Apache vhost.
+1. Set up the site as an Apache vhost. An example can be seen in
+   `docs/apache.conf.example`.
 
-12. Create a clean database in MySQL.
+1. Create a clean database in MySQL.
 
-13. Rename `example_creds.php` to `creds.php` and enter your DB info, as well as
-admin info for the initial admin user seed. Additionally, do the same for
-`example_smtp.php` if you want to enable email functionality (required for
-registering new users).
+1. Rename `server/config/example_creds.php` to `server/config/creds.php` and
+enter your DB info, as well as admin info for the initial admin user seed.
+Additionally, do the same for `example_smtp.php` if you want to enable email
+functionality (required for registering new users).
 
-12. Run `php artisan migrate` and `php artisan db:seed` from the root directory to
-set up the database.
+1. Run `make rebuild-db` to get the database migrated and seeded with data.
 
-13. Install and run elasticsearch. [See here for installation](elasticsearch).
+1. Install and run elasticsearch. [See here for installation](elasticsearch).
   1. Additionally, run `curl -XPUT http://localhost:9200/madison` when ES is running
   to create a new index.
 
-14. Copy the `.env.example` file to `.env` and set the appropriate variables.
+1. Copy the `server/.env.example` file to `server/.env` and set the appropriate variables.
   1. If you don't have one already, set up an account at [MailTrap](mailtrap)
   and set the `MAIL_USERNAME` and `MAIL_PASSWORD` variables to what it provides
   for your test inbox. This is needed for user signup.
 
 ### Note
 
-When editing any .js files, make sure to run `grunt watch` so that it can watch for and
-re-minify (among other things) the main build/app.js and build/app.map files.
+When editing any .scss files, make sure to run `make watch` so that it can watch for and
```

Comment:
Only for Sass files? Does it not do anything with JS? Do we need anything to watch JS files during development?